Extensions/xkit_updates.js

```diff
@@ -468,6 +468,39 @@ XKit.extensions.xkit_updates = new Object({
 
 	},
 
+	shouldUpdate: function(versionCurrent, versionRemote) {
+		if (versionRemote.major > versionCurrent.major) {
+			return true;
+		}
+		if ((versionRemote.minor > versionCurrent.minor) && (versionRemote.major <= versionCurrent.major)) {
+			return true;
+		}
+		if ((versionRemote.patch > versionCurrent.patch) && (versionRemote.major <= versionCurrent.major) && (versionRemote.minor <= versionCurrent.minor)) {
```

Comment:
same thing as above `versionRemote.major <= versionCurrent.major` should be `versionRemote.major === versionCurrent.major` and `versionRemote.minor <= versionCurrent.minor` should be `versionRemote.minor === versionCurrent.minor`